### DISCUSSION TITLE
Request checkbox to display deleted folders, files in the UI

### DISCUSSION THREAD

**Discussion by Capstan** [community/community#19296] [Other Feature Feedback, Questions, & Ideas]
It would be handy if there were a setting or a UI element like

> ☑️ Display deleted folders and files

so that when I'm in the Code tab, I get entries, perhaps marked in ~strikeout~ or red or both that indicated that a directory or file was deleted, e.g.,

> 📄 `my_deleted_file.txt` chore(main): clean up things that need deleting (#1234) deleted 25 days ago

I am often trying to figure out when something got deleted, and having to go back to peering at commit logs in a directory or reverting to using the cli, and a quick display of the things that _used_ to be there would be super useful.

**Comment by webAKAsh** [comment]
A recycle Bin tab can be created where all the deleted stuff can be handled in a table.
